projects/tree/src/components/tree-layout/components/tree-node-description/tree-node-description.component.scss

```
@@ -0,0 +1,9 @@
+@use 'functions' as *;
+@use 'mixins' as *;
+
+:host {
+  display: block;
+  @include ellipsis-text-overflow;
+  @include font-size-xxs();
```

```suggestion
  @include font-size-xxs;
```

Так как выше без скобок